.github/workflows/benchmark1.yaml

```diff
@@ -124,7 +124,7 @@ jobs:
       - name: E2E target cloud (modelservice, fmperf)
         env:
           LLMDBENCH_HF_TOKEN: ${{ secrets.LLMDBENCH_HF_TOKEN }}
-        run: ./setup/e2e.sh -c cicd -t modelservice --deep -l fmperf -w sanity_short-input
+        run: ./setup/e2e.sh -c cicd -t modelservice --deep -l fmperf -w sanity_short-input.yaml
```

Comment:
Why is `.yaml` required here but not in the two edits before this?

Reply:
Not "required" but actually "allowed". The idea is to test both cases, with and without `.yaml`

Reply:
Got it makes sense!